### ISSUE TITLE
Unique Form IDs

### ISSUE THREAD

**Issue by cdils** [kraftbj/genesis-enews-extended#113]
When multiple instances of the Enews widget are used on the same page, they each get the same form ID assigned:

`<form id="subscribe" ... `

For accessibility purposes, each form on a page should have a unique ID.
